src/window/gui/Gui.cpp

```diff
@@ -210,6 +210,7 @@ void Gui::ImGuiBackendInit() {
             ImGui_ImplGX2_Init();
             break;
 #else
+#ifdef ENABLE_OPENGL
```

Comment:
having this right after the else from wii u makes for a bit of complex nesting, might be nice to add some comments on the endifs to make it easier to tell where these start/end